ios/RiveReactNativeViewManager.m

```diff
@@ -6,6 +6,7 @@ @interface RCT_EXTERN_MODULE(RiveReactNativeViewManager, RCTViewManager)
 RCT_EXPORT_VIEW_PROPERTY(resourceName, NSString)
 RCT_EXPORT_VIEW_PROPERTY(url, NSString)
 RCT_EXPORT_VIEW_PROPERTY(fit, NSString)
+RCT_EXPORT_VIEW_PROPERTY(layoutScaleFactor, NSNumber)
```

Comment:
I'm assuming this has something to do with using NSNumber over Double?

Reply:
Yup! In one of the other comments I linked to the reference docs